os/drivers/kernel_test/os_test_drv.c

```diff
@@ -27,8 +27,7 @@
 #include <tinyara/fs/fs.h>
 #include <tinyara/kernel_test_drv.h>
 #include <tinyara/sched.h>
-#include "signal/signal.h"
-#include "kernel_test_proto.h"
+#include "os_test_proto.h"
```

Comment:
Also, need to rename all api's and data structures.

Reply:
done